modules/images-configuration-image-registry-settings-hcp.adoc

```diff
@@ -96,17 +96,21 @@ Audit Log Forwarding:       Disabled
 External Authentication:    Disabled
 Etcd Encryption:            Disabled
 Registry Configuration:
- - Allowed Registries:     <allowed_registry> <1>
- - Insecure Registries:    <insecure_registry> <2>
- - Allowed Registries for Import: <3>
-    - Domain Name:          <domain_name> <4>
-    - Insecure:             true <5>
+ - Allowed Registries:     <allowed_registry> <1> <2>
+ - Insecure Registries:    <insecure_registry> <3>
+ - Allowed Registries for Import: <4>
+    - Domain Name:          <domain_name> <5>
+    - Insecure:             true <6>
+ - Additional Trusted CA: <7>
+    - <registry_name> : REDACTED
 ----
 <1> `Allowed Registries`: A comma-separated list of registries for which image pull and push actions are allowed.
-<2> `Insecure Registries`: A comma-separated list of registries which do not have a valid TLS certificate or only support HTTP connections.
-<3> `Allowed Registries for Import`: Limits the container image registries from which normal users can import images. The format should be a comma-separated list of `domainName:insecure`.
-<4> `domainName`: Specifies a domain name for the registry.
-<5> `insecure`: Indicates whether the registry is secure or insecure.
+<2> `Blocked Registries`: A comma-separated list of registries for which image pull and push actions are blocked. Parameters `allowedRegistries`, `blockedRegistries` are mutually exclusive.
+<3> `Insecure Registries`: A comma-separated list of registries which do not have a valid TLS certificate or only support HTTP connections.
+<4> `Allowed Registries for Import`: Limits the container image registries from which normal users can import images. The format should be a comma-separated list of `domainName:insecure`.
+<5> `domainName`: Specifies a domain name for the registry.
+<6> `insecure`: Indicates whether the registry is secure or insecure.
+<7> `Additional Trusted CA`: A JSON file containing the registry hostname as the key, and the PEM-encoded certificate as the value, for each additional registry CA to trust.
```

Comment:
It should be same with cluster edition.

Reply:
applied

Reply:
The Platform Allowlist item is not shown in cluster description  after cluster creation